src/components/Banner/SpringSurgeBanner.tsx

```diff
@@ -13,22 +14,28 @@ const trackClick = () => {
 };
 
 const BannerInner: React.FC = () => {
-  const SeveralStatesLink = (
+  const usaAggregation = AggregationsJSON['00001'];
+  const { totalVaccinationsInitiated } = usaAggregation;
+  const millionVaccinations = Math.round(totalVaccinationsInitiated / 1000000);
+
+  const ManyStatesLink = (
     <HashLink
       to="#explore-hospitalizations"
       onClick={trackClick}
       scroll={(element: HTMLElement) => scrollWithOffset(element, -80)}
     >
-      several states
+      many states
```

Comment:
I can revert if you're not a fan, but about half of the states that have rising cases also have rising hospitalizations so "many" feels appropriate.

Reply:
I like it. "Many" does feel appropriate.